src/prpy/planning/ompl.py

```diff
@@ -217,16 +219,16 @@ def _Plan(self, robot, goal=None, tsrchains=None, timelimit=None,
         params.SetExtraParameters(extraParams)
 
         if (not continue_planner) or (not self.setup):
-            self.planner.InitPlan(robot, params)
-            self.setup = True
+            planner.InitPlan(robot, params)
+            setup = True
 
         # Bypass the context manager since or_ompl does its own baking.
         env = robot.GetEnv()
         robot_checker = self.robot_checker_factory(robot)
         options = robot_checker.collision_options
         with CollisionOptionsStateSaver(env.GetCollisionChecker(), options):
             traj = RaveCreateTrajectory(env, 'GenericTrajectory')
-            status = self.planner.PlanPath(traj, releasegil=True)
+            status = planner.PlanPath(traj, releasegil=True)
```

Comment:
This should be wrapped in a `RobotStateSaver` that saves `robot`'s `LinkTransformation`s.